src/phoneField/index.tsx

```diff
@@ -33,27 +35,27 @@ interface PhoneFieldProps {
   readonly countriesWhiteList?: string[]
   readonly customCountriesName?: customCountriesName
   readonly isInline?: boolean
-  readonly onChange?: (obj: PhoneFieldOnChangeParameters) => void
 }
 interface PhoneFieldState {
   countriesData: mappedCountriesPhoneData[]
   countriesWhiteList: string[]
+  phoneRegion: string[]
   completePhoneNumber: string
   // Make the type of [key: string]: string etc... assignable for setState()
   [key: string]: any
 }
 
 const allCountriesPhoneData: allCountriesPhoneData[] = countryTelData.allCountries
+const allCountriesCode: allCountriesCode = countryTelData.allCountryCodes
```

Comment:
Can't you destructure countryTelData directly from the import?

Reply:
Done